tf/main.tf

```diff
@@ -26,3 +26,16 @@ resource "aws_dynamodb_table" "backend-tf-locks-table" {
   }
   tags = local.default_tags
 }
+
+resource "aws_s3_object" "cirrus-versions" {
+  bucket = aws_s3_bucket.backend-tf-state-bucket.id
```

Comment:
should we put this in the cumulus internal bucket?

Reply:
Me and Rohan talked about this. That is originally where I had it. CIRRUS isn't inherently apart of the cumulus. We decided on the TF bucket.

Reply:
IIRC, the logic was that CIRRUS is used a `scaffolding` for Terraform.

Reply:
Yea the problem with the internal bucket is that it's part of the `daac` module in CIRRUS-DAAC. That creates a whole host of problems like having to carefully manage the order of deployment, and managing inconsistencies if the deploy fails before the version file can be uploaded. Not to mention that we can't actually be sure that the internal bucket exists at all since a DAAC has full control over their CIRRUS-DAAC fork and could have removed it or renamed it.

It makes a lot more sense I think to write it in the `tf` step into a resource that is created in that step.

Reply:
The only thing I am concerned about is that is that bucket should be kept very secure as it could contain sensitive information in the remote state files and its ment to be used for remote statefile management. Putting extra files in there feels a bit weird. But if no one else feels that way, fine with me.

Reply:
could we make it output that can be read from the statefile? What is the intended way to get this information?